docker-compose.yml

```diff
@@ -119,6 +119,7 @@ services:
       REPOSITORY_GITHUBUSERTOKEN: ${SGOV_SERVER_REPOSITORY_GITHUBUSERTOKEN}
       USER_NAMESPACE: https://slovník.gov.cz/uživatel/
       USER_CONTEXT: https://slovník.gov.cz/uživatel
+      FEATURE_DEMO: "false"
```

Comment:
I guess this works, but it may be better to take the value from env var instead of hardcoding it here.

Reply:
I like to add the default value to the configuration, so I know how it works when I don't set it:
```
${FEATURE_DEMO:-false}
````

OK ? Or should I replace it with just 
```
${FEATURE_DEMO}
```